Silicon/Intel/IntelSiliconPkg/Feature/Flash/SpiFvbService/SpiFvbServiceMm.c

```diff
@@ -25,12 +26,12 @@
 **/
 VOID
 InstallFvbProtocol (
-  IN  EFI_FVB_INSTANCE               *FvbInstance
+  IN  EFI_FVB_INSTANCE  *FvbInstance
   )
 {
-  EFI_FIRMWARE_VOLUME_HEADER            *FvHeader;
-  EFI_STATUS                            Status;
-  EFI_HANDLE                            FvbHandle;

```

Comment:
such changes are for uncrustify coding style.

Reply:
We have a fork of edk2-platforms and have held off on using uncrustify since tianocore/edk2-platforms has not converted files over like tianocore/edk2.

I'd suggest doing one batch operation of applying uncrustify to all files in the repo and then there will be no formatting only diffs like this in the future. In the meantime, I guess that'd also mean to suggest dropping the formatting from this patch until that happens.

Reply:
Also, in case it wasn't obvious, the whole repo can be converted over with a few commands.

Reply:
ok. I think I can revert uncurstify changes for now and focus on bug fix changes.